packages/xstate-graph/test/graph.test.ts

```diff
@@ -61,7 +60,8 @@ describe('@xstate/graph', () => {
     }
   };
 
-  const lightMachine = Machine({
+  const lightMachine = createMachine({
+    predictableActionArguments: true,
```

Comment:
Those warnings in the test output are annoying but I think it's better to avoid using this flag in tests right now. The reason is that this flag doesn't exist in v5 (it's just the default) and we don't want to end up cleaning up those flags when merging to `next` (+we avoid conflicts by not using this flag)